### ISSUE TITLE
Remove/filter bug events from the testnet

### ISSUE THREAD

**Issue by ippudo** [bodhiproject/bodhi-ui#924]
![wocao](https://user-images.githubusercontent.com/9134404/45115019-7d4cbf00-b103-11e8-9ad5-3164a9d91021.png)

There're 4 events created months ago with a wrong timestamp. I'm wondering if we could remove or filter them.

**Comment by dwalintukan** [comment]
@ippudo not possible. once its on the blockchain its there forever.

**Comment by ippudo** [comment]
Yeah I guess so. So I think we may need to filter the exists bug events?

**Comment by dwalintukan** [comment]
no need. as long as this is confined to testnet then its fine. there will be trash events on testnet. none on mainnet though right?
